README.md

```diff
@@ -4,18 +4,24 @@ Configuration instructions, files and scripts for deploying PeachCloud.
 
 _Work in progress._
 
-**Network**
+### Prerequisite Steps
```

Comment:
could consider having the rerequisite steps link to the relevant peach-docs, or vice versa, 
so that we don't have duplicate docs that could go out of sync with each other 
(stale comments sometimes worse than no comments)

Reply:
Hmm yeah I was thinking the same thing. On the one hand, it seems best to avoid unnecessary duplication wherever possible. On the other hand, I quite like the idea of having the devdocs contain all necessary information (then it could be printed as a PDF book and still be useful). I'll give it some thought.

One option would be for the peach-config README to be replicated to the Quick Deployment devdocs page on each main branch update. This would also be useful for the microservice docs. I'll see if I can figure out how to automate that.